go.sum

```diff
@@ -99,6 +96,8 @@ github.com/prometheus/common v0.6.0/go.mod h1:eBmuwkDJBwy6iBfxCBob6t6dR6ENT/y+J+
 github.com/prometheus/procfs v0.0.0-20181005140218-185b4288413d/go.mod h1:c3At6R/oaqEKCNdg8wHV1ftS6bRYblBhIjjI8uT2IGk=
 github.com/prometheus/procfs v0.0.2/go.mod h1:TjEm7ze935MbeOT/UhFTIMYKhuLP4wbCsTZCD3I8kEA=
 github.com/prometheus/procfs v0.0.3/go.mod h1:4A/X28fw3Fc593LaREMrKMqOKvUAntwMDaekg4FpcdQ=
+github.com/qmuntal/notation-go v0.7.0-alpha.1.0.20220429152921-f0636e164767 h1:jgS3O1+FduH9ywKWadFSFcnNIcdbuJmqUV2RFMDCv30=
+github.com/qmuntal/notation-go v0.7.0-alpha.1.0.20220429152921-f0636e164767/go.mod h1:B/JbgikwvLoD9gFeWYhFtxIEozRoFAs9q31mj3vz1i4=
```

Comment:
Assuming this will be updated once plugin manager updates are merged to main.

Reply:
I've changed notation-go reference so it points to `https://github.com/notaryproject/notation@main`, as the necessary functionality has already been merged.